bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZLLGTo2L2j_5f_NLO_FXFX/ZLLGTo2L2j_5f_NLO_FXFX_customizecards.dat

```diff
@@ -0,0 +1,5 @@
+#put card customizations here (change top and higgs mass for example)
+set param_card mass 6 172.5
+set param_card yukawa 6 172.5
+set param_card mass 25 125.0
+
```

Comment:
this card may not needed, the higgs mass is 125 in the model, and the top mass will not affect your process here